### ISSUE TITLE
[Accessibility]A11y_VSCode_ViewExtensions_ScreenReader:ScreenReader focus does not go to disabled control.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#52616]
**Environment Details**:
VSCode Version : 1.24.1
OS Version : WIN10
**Additional Details:**
ScreenReaderNVDA[2018.2],JAWS[18.0.5038]MAS1.3.1,MACOS High Sierra 10.13.5 Beta (17F70a)
**Repro Steps:**
1. Launch VS Code.
2. Navigate to View(Alt+V) menu option and select "Extensions" from dropdown.
3. Navigate to more button then navigate to Clear input button.
4. Observe what screenreader reads.

**Actual:**
 Screenreader does not read the statefor  "Clear extension input"  button  as disabled. 
Same issue observed in Jaws also.
**Expected:**
Screenreader  should read the  "Clear extension input" button as disabled.

Same issue observed in MAC also.
**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
[https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx](url)

**User Impact:**
User unable to know what is present on the screen.
**MAS Reference:**
[https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}](url)

Does this issue occur when all extensions are disabled?: Yes
**Attachment For Reference**

[[Accessibility]A11y_VSCode_ViewExtension_NVDA1.pptx](https://github.com/Microsoft/vscode/files/2123856/Accessibility.A11y_VSCode_ViewExtension_NVDA1.pptx)

**Comment by isidorn** [comment]
Dup of https://github.com/Microsoft/vscode/issues/52577
